cmd/aks-periscope/aks-periscope.go

```diff
@@ -49,6 +50,7 @@ func main() {
 	collectors = append(collectors, dnsCollector)
 	collectors = append(collectors, kubeObjectsCollector)
 	collectors = append(collectors, networkOutboundCollector)
+	collectors = append(collectors, containerDLogsCollector)
```

Comment:
💡 Thanks for this, I have few questions to understand more about this work, what is the use-case with vanilla `aks-periscope` tool run with this please? can you add an example in description:

Few questions: 

So currently this tool runs on AKS Cluster, this work enables this `critical` command with in collector to always collect `containers` logs? 

Will there be any scenario where user would not want this auto enabled?

Reply:
Actually, if you run the current aks-periscope on a public AKS cluster, it will NOT collect any container logs, as AKS has switched from docker to contianerd as default container runtime. I think this new collector should replace the existing "docker" container collecter. So `crictl` is the CLI tool to interact with containerd (precisely cri). Please read this article for details: https://www.tutorialworks.com/difference-docker-containerd-runc-crio-oci/